NoFences/Model/FenceInfo.cs

```diff
@@ -13,15 +13,24 @@ public class FenceInfo
 
         public int PosY { get; set; }
 
+        /// <summary>
+        /// Gets or sets the DPI scaled window width.
+        /// </summary>
         public int Width { get; set; }
 
+        /// <summary>
+        /// Gets or sets the DPI scaled window height.
+        /// </summary>
         public int Height { get; set; }
 
         public bool Locked { get; set; }
 
         public bool CanMinify { get; set; }
 
-        public int TitleHeight { get; set; } = 35;
+        /// <summary>
+        /// Gets or sets the logical window title height.
+        /// </summary>
+        public int LogicalTitleHeight { get; set; } = 35;
```

Comment:
Please don't change the names of the model, as it is serialized to XML. Changing the names would break compatibility with old versions.